src/clj/puppetlabs/services/ca/certificate_authority_core.clj

```diff
@@ -95,6 +95,20 @@
           (rr/status 304)
           (rr/content-type "text/plain")))))
 
+(schema/defn handle-put-certificate-revocation-list!
+  [incoming-crl-pem :- InputStream
+   {:keys [cacrl cacert infra-crl-path enable-infra-crl]} :- ca/CaSettings]
+  (locking crl-write-serializer
+    (try
+      (ca/update-crls incoming-crl-pem cacrl cacert)
+      (when enable-infra-crl
+        (ca/update-crls incoming-crl-pem infra-crl-path cacert))
```

Comment:
If this endpoint is used to update our internal CRL, it will do the wrong thing with this infra CRL (i.e. set it to be the same as the full CRL). Do we want to try to avoid that somehow?